ios/OwrHelpers/OpenWebRTCNativeHandler.m

```diff
@@ -0,0 +1,836 @@
+//
+//  Redistribution and use in source and binary forms, with or without modification,
+//  are permitted provided that the following conditions are met:
+//
+//  1. Redistributions of source code must retain the above copyright notice, this
+//  list of conditions and the following disclaimer.
+//
+//  2. Redistributions in binary form must reproduce the above copyright notice, this
+//  list of conditions and the following disclaimer in the documentation and/or other
+//  materials provided with the distribution.
+
+//  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+//  ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+//  WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
+//  IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+//  INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT
+//  NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+//  PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+//  WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+//  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY
+//  OF SUCH DAMAGE.
+//
+
+#import "OpenWebRTCNativeHandler.h"
+
+
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+#include "owr.h"
+#include "owr_audio_payload.h"
+#include "owr_audio_renderer.h"
+//#include "owr_local.h"
+#include "owr_media_session.h"
+//#include "owr_transport_agent.h"
+#include "owr_video_payload.h"
+#include "owr_video_renderer.h"
+#include "owr_window_registry.h"
+
+#include <gio/gio.h>
+//#include <json-glib/json-glib.h>
+//#include <libsoup/soup.h>
+#include <string.h>
+
+#define SELF_VIEW_TAG "self-view"
+#define REMOTE_VIEW_TAG "remote-view"
+
+OwrVideoRenderer *renderer;
+
+static OpenWebRTCNativeHandler *staticSelf;
+
+@implementation OpenWebRTCNativeHandler
+
+- (instancetype)init
+{
+    if (self = [super init]) {
+        owr_init();
+        staticSelf = self;
+    }
+    return self;
+}
+
+- (void)setSelfView:(OpenWebRTCVideoView *)selfView
+{
+    owr_window_registry_register(owr_window_registry_get(), SELF_VIEW_TAG, (__bridge gpointer)(selfView));
+}
+
+- (void)setRemoteView:(OpenWebRTCVideoView *)remoteView
+{
+    owr_window_registry_register(owr_window_registry_get(), REMOTE_VIEW_TAG, (__bridge gpointer)(remoteView));
+}
+
+- (void)handleOfferReceived:(NSString *)offer
+{
+
+}
+
+- (void)handleRemoteCandidateReceived:(NSString *)candidate
+{
+
+}
+
+- (void)startGetCaptureSources:(OwrMediaType)types
+{
+    owr_get_capture_sources(types, (OwrCaptureSourcesCallback)got_local_sources, NULL);
+}
+
+#pragma mark - C stuff
+
+static GList *local_sources, *renderers;
+static OwrTransportAgent *transport_agent;
+//static gchar *session_id, *peer_id;
+static gchar *peer_id;
+//static guint client_id;
```

Comment:
Same here and everywhere. If headers, variables, code is unused and you don't want to keep it for a specific reason, please remove it.